libpod/container_api.go

```diff
@@ -705,3 +705,19 @@ func (c *Container) Sync() error {
 
 	return nil
 }
+
+// Restart restarts a running container and takes a given timeout in uint
+func (c *Container) Restart(timeout uint) error {
```

Comment:
Call this RestartWithTimeout(), our standard Stop() doesn't take a timeout

Reply:
fixed